fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchQuantityTest.java

```diff
@@ -170,163 +187,195 @@ public void testSearchQuantity_Quantity_NoDisplayUnit() throws Exception {
-        
-//        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "lt2||lte");      // <= 3 may be < 2
-        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "gt2||lte");      // <= 3 may be > 2
-        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "lt4||lte");      // <= 3 may be < 4 
+
+        assertSearchDoesntReturnSavedResource("Quantity-lessThanOrEqual", "lt2||lte");      // <= 3 may be < 2
+        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "gt2||lte"); // <= 3 may be > 2
+        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "lt4||lte"); // <= 3 may be < 4 
         assertSearchDoesntReturnSavedResource("Quantity-lessThanOrEqual", "gt4||lte"); // <= 3 is not > 4
-        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "le3||lte");      // <= 3 is <= 3
-        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "ge3||lte");      // <= 3 may be >= 3
+        
+        // As we have added implict ranges to the prefix processing, we need to add precision 
+        // >= 3 may be <= 3 uses precision and bounding for the range. 
+        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "le3||lte");
+        assertSearchReturnsSavedResource("Quantity-lessThanOrEqual", "ge3||lte"); // <= 3 may be >= 3
     }
-    
+
     @Test
     public void testSearchQuantity_Quantity_GreaterThanOrEqual() throws Exception {
         assertSearchDoesntReturnSavedResource("Quantity-greaterThanOrEqual", "2||gte");
         assertSearchReturnsSavedResource("Quantity-greaterThanOrEqual", "3||gte");
-//        assertSearchReturnsSavedResource("Quantity-greaterThanOrEqual", "4||gte");
-        
+        assertSearchDoesntReturnSavedResource("Quantity-greaterThanOrEqual", "4||gte");
+
         assertSearchDoesntReturnSavedResource("Quantity-greaterThanOrEqual", "lt2||gte"); // >= 3 is not < 2
-        assertSearchReturnsSavedResource("Quantity-greaterThanOrEqual", "gt2||gte");      // >= 3 may be > 2
-        assertSearchReturnsSavedResource("Quantity-greaterThanOrEqual", "lt4||gte");      // >= 3 may be < 4 
-//        assertSearchReturnsSavedResource("Quantity-greaterThanOrEqual", "gt4||gte");      // >= 3 may be > 4
-        assertSearchReturnsSavedResource("Quantity-greaterThanOrEqual", "le3||gte");         // >= 3 may be <= 3
-        assertSearchReturnsSavedResource("Quantity-greaterThanOrEqual", "ge3||gte");         // >= 3 is >= 3
+        assertSearchReturnsSavedResource("Quantity-greaterThanOrEqual", "gt2||gte"); // >= 3 may be > 2
+        assertSearchReturnsSavedResource("Quantity-greaterThanOrEqual", "lt4||gte"); // >= 3 may be < 4 
+        assertSearchDoesntReturnSavedResource("Quantity-greaterThanOrEqual", "gt4||gte");      // >= 3 may be > 4
+        
+        // As we have added implict ranges to the prefix processing, we need to add precision 
+        // >= 3 may be <= 3 uses precision and bounding for the range. 
```

Comment:
I suggest removing this.  At least the first line and probably the second as well.